scripts/helper_functions.sh

```diff
@@ -216,12 +218,19 @@ function install_plugins {
     local path=$1
     # Commenting the plugin install as Azure Cache for Redis is not currently being deployed
     # Uncomment or replace with appropriate plugin when Redis Cache infrastructure is deployed again
-    # wp plugin install w3-total-cache --path=$path --allow-root
-    # wp plugin activate w3-total-cache --path=$path --allow-root
     wp plugin activate akismet --path=$path --allow-root
     chown -R www-data:www-data $path
 }
 
+function install_plugin_w3_total_cache {
+  local wpPath="$1"
+  wp plugin install w3-total-cache --path=$wpPath --allow-root
+  wp plugin activate w3-total-cache --path=$wpPath --allow-root
+
+  # execute (not import) w3tc setup script
```

Comment:
why is it not important to execute setup script ?

Reply:
trying to understand the comment.

Reply:
it says execute script in its own process rather than importing it. If we import, all the variables will be added.